docker/versions.sh

```diff
@@ -23,8 +23,8 @@ export CTRL_OODS_GIT=tags/2.2.0
 export DM_CONFIG_CC_GIT=tags/1.0.0
 export DM_CCARCHIVER_GIT=tags/1.2.0
 #
-export DM_CONFIG_CATCHUP_GIT=tickets/DM-24077
-export DM_CATCHUPARCHIVER_GIT=tickets/DM-24077
+export DM_CONFIG_CATCHUP_GIT=master
+export DM_CATCHUPARCHIVER_GIT=master
```

Comment:
Once this is deployed, you'll have to tag the release version and and that as is done for the other containers.  That way we'll know which version of the catchup archiver we're running.  It's fine for now though.